airbyte-integrations/connectors/destination-pinecone/integration_tests/pinecone_integration_test.py

```diff
@@ -4,33 +4,54 @@
 
 import json
 import logging
+import time
 
-import pinecone
 from airbyte_cdk.destinations.vector_db_based.embedder import OPEN_AI_VECTOR_SIZE
 from airbyte_cdk.destinations.vector_db_based.test_utils import BaseIntegrationTest
 from airbyte_cdk.models import DestinationSyncMode, Status
 from destination_pinecone.destination import DestinationPinecone
 from langchain.embeddings import OpenAIEmbeddings
 from langchain.vectorstores import Pinecone
+from pinecone import Pinecone as PineconeREST
+from pinecone import PineconeException
+from pinecone.grpc import PineconeGRPC
 
 
 class PineconeIntegrationTest(BaseIntegrationTest):
     def _init_pinecone(self):
-        pinecone.init(api_key=self.config["indexing"]["pinecone_key"], environment=self.config["indexing"]["pinecone_environment"])
-        self.pinecone_index = pinecone.Index(self.config["indexing"]["index"])
-
+        self.pc = PineconeGRPC(api_key=self.config["indexing"]["pinecone_key"])
+        self.pinecone_index = self.pc.Index(self.config["indexing"]["index"])
+        self.pc_rest = PineconeREST(api_key=self.config["indexing"]["pinecone_key"])
+        self.pinecone_index_rest = self.pc_rest.Index(name=self.config["indexing"]["index"])
+    
+    def _wait(self):
+        print("Waiting for Pinecone...", end='', flush=True)
+        for i in range(15):
+            time.sleep(1)
+            print(".", end='', flush=True)
+        print()  # Move to the next line after the loop
+    
     def setUp(self):
         with open("secrets/config.json", "r") as f:
             self.config = json.loads(f.read())
         self._init_pinecone()
+        # self.pinecone_index.delete(delete_all=True)
```

Comment:
Should we remove this commented code, or (if sometimes useful) add a comment above about when to toggle it to uncommented?

If removing, we can apply this suggestion:

```suggestion
    def setUp(self):
        with open("secrets/config.json", "r") as f:
            self.config = json.loads(f.read())
        self._init_pinecone()
```

Reply:
we can delete the code. we do this in teardown, so kind of unnecessary here.